admin_guide/diagnostics_tool.adoc

```diff
@@ -115,3 +115,54 @@ A client with *cluster-admin* access available (for any user, but only the
 current master) should be able to diagnose the status of infrastructure such as
 nodes, registry, and router. In each case, running `oc adm diagnostics` looks
 for the client configuration in its standard location and uses it if available.
+
+[[additional-cluster-health-checks]]
+== Additional Diagnostic Checks via Ansible
+
+Some additional diagnostic checks are available through the https://hub.docker.com/r/openshift/openshift-ansible[openshift-ansible]
+Docker image.
+
+These checks belong to a diagnostic task meant to be run against the Ansible inventory file for a deployed OpenShift cluster.
+They can report common OpenShift-specific problems for the current deployment.
+
+To disable specific checks, include the variable `openshift_disable_check` with a
+comma-delimited list of check names in your inventory file. For example:
+
+----
+openshift_disable_check=ovs_version,etcd_volume
+----
+
+The following table describes available health checks that can be run on a deployed OpenShift cluster.
+
+[[diagnostic-checks]]
+.Diagnostic Checks
+[options="header"]
+|===
+
+|Check Name |Purpose
```

Comment:
Are we willing to keep this table up-to-date?
I'm concerned about having to manually keep this in sync with code in openshift-ansible.